x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/EqlFunctionRegistry.java

```diff
@@ -37,6 +38,8 @@ public EqlFunctionRegistry() {
                 def(EndsWith.class, EndsWith::new, "endswith"),
                 def(IndexOf.class, IndexOf::new, "indexof"),
                 def(Length.class, Length::new, "length"),
+                def(Match.class, Match::new, "match"),
+                def(Match.class, Match::new, "matchLite"),
```

Comment:
This function is just an old alias for `match` that was kept for compatibility. Should we drop?

Reply:
I think this is the answer for the question you asked [here](https://github.com/elastic/elasticsearch/pull/55182#discussion_r408456016). The definition of `def` accepts more values for the third argument. Everything past the first value is an "alias". I would try with `matchLite` definition as an alias. There are plenty of examples in [SqlFunctionRegistry](https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/SqlFunctionRegistry.java).

Reply:
fixed in 521b3f9d137